docs/index.html

```diff
@@ -523,6 +538,17 @@ <h5>SSH Agent integration</h5>
             Instead of letting the <code>IdentityFile</code> directive point to a private key file, let it point to
             your public key file. The SSH Agent will use the provided information to select the correct private key.
         </dd>
+
+        <dt id="faq-ssh-agent-git-bash"><a href="#faq-ssh-agent-git-bash">How do I use KeePassXC SSH Agent integration with Git (Bash) on Windows?</a></dt>
```

Comment:
This should be expanded to include how to change the `ssh.exe` used by Git to Windows 10 OpenSSH as that's the most native way.

Reply:
Ah, it's a little implicit I grant you that! There's three ways to achieve that: have Git for Windows 10+ use the Windows supplied version when prompted during installation of Git, modify the Git Bash `$PATH` by prepending the Windows OpenSSH Path – that makes Git for Windows prefer it to its own (if installed) – or set the `GIT_SSH_COMMAND` variable. The last option will _only_ work for Git, however, no other binary in the Git Bash will sadly pick it up leading to some weird edge cases. Shall I include this in the FAQ?

Reply:
I added an additional sentence explaining `$PATH` and `$GIT_SSH_COMMAND` options.